packages/story-editor/src/app/story/useStoryReducer/reducers/copyElementById.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import objectPick from '../../../../utils/objectPick';
+
+export const ATTRIBUTES_TO_COPY = [
+  'background',
+  'backgroundColor',
+  'backgroundTextMode',
+  'border',
+  'borderRadius',
+  'content',
+  'flip',
+  'font',
+  'fontSize',
+  'lineHeight',
+  'lockAspectRatio',
+  'opacity',
+  'overlay',
+  'padding',
+  'rotationAngle',
+  'textAlign',
+];
```

Comment:
💭  The old copy/paste functionality depended on the `copyableAttributes` objects that were in each element folder. That was confusing, and different for each element type because each element has different properties that can be updated

I removed the old confusing code and centralized all properties that may be copy and pasted here. If an element has any of these properties, they may be copied and pasted to a different element of the same type.